modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/list/List.tsx

```diff
@@ -194,23 +194,25 @@ const ListItem = forwardRef<HTMLLIElement, any>(
 );
 
 const ListItemOptionalDropTarget = ({
+	className,
 	clayListItemProps,
 	item,
 	items,
 	onItemSelectionChange,
 	schema,
 }: {
+	className?: string;
 	clayListItemProps?: object;
 	item: any;
 	items: any[];
 	onItemSelectionChange: Function;
 	schema: IListSchema;
 }) => {
-	const {className, dropRef} = useFDSDrop({item});
+	const {className: classNameFDSDrop, dropRef} = useFDSDrop({item});
```

Comment:
Generally, in a name, we put specific descriptior as prefix, more general descriptor as suffix. For example, in `clayListItemProps`, `props` is very general, `itemProps` is more specific, `listItemProps` is even more specific, `clayListItemProps` is very specific.

In your case, the variable does not contain a `drop` or a `fdsDrop`. What would that even be? And what is this `className` variant of a `drop`?

Please rename variable to something like `dropClassName`. `FDS` in name is not needed, as this is internal variable to FDS module, so everything is implicitly related to FDS.